Cargo.toml

```diff
@@ -58,7 +59,7 @@ usbd-ccid = { version = "0.2.0", features = ["highspeed-usb"]}
 rand = "0.8.5"
 
 [features]
-default = []
+default = ["virt"]
```

Comment:
Is this intended?

Reply:
Nope, I set it to get rust-analyzer to give me completions but it shouldn't have made it to a commit.